### ISSUE TITLE
Learn How to Deploy IDURAR ERP CRM (Create Node.js APi and crud React app in. 1 hour)

### ISSUE THREAD

**Issue by salahlalami** [idurar/idurar-erp-crm#713]
🔥 I would like to invite you for a weekly free IDURAR Webinar (Node.js React.js course), where you learn how to deploy IDURAR on cloud , and create a new api and new crud app with IDURAR in just one hour ?
Please fill this form if you are interested : [https://forms.gle/qz2YZ3xQFQ77bGhS8](https://forms.gle/qz2YZ3xQFQ77bGhS8)

The Webinar will be this Wednesday at 1pm GMT.

**Comment by johnvan7** [comment]
I'm interested but I am not available on Wednesday 😢. Will it be recorded?

**Comment by salahlalami** [comment]
@johnvan7 , yes i will record it and put it my youtube channel : [https://www.youtube.com/@lalamisdn](https://www.youtube.com/@lalamisdn)

**Comment by abhijadon** [comment]
sir api kaise post kre localhost vs code thunder client se y message show ho rha h {
  "success": false,
  "result": null,
  "message": "No authentication token, authorization denied.",
  "jwtExpired": true
}

**Comment by Prabhat-6789** [comment]
@salahlalami, how can I join webinar? will you provide link? I have already fill google form.

**Comment by dananjaya6005** [comment]
can we host it on render ?

**Comment by Muhammad-Mubashar516** [comment]
hlo,i am contribute it,so if you give me permision!
